reference/features/search_parameters.md

```diff
@@ -224,60 +249,47 @@ And you would get the following response:
 
 ## Attributes to retrieve
 
-Attributes to **display** in the returned documents.
+**Parameter**: `attributesToRetrieve`
+**Expected value**: an array of `attribute`s or `["*"]`
+**Default value**: `["*"]`
 
-`attributesToRetrieve=<Attribute>,<Attribute>,...`
+Configures which attributes will be displayed in the returned documents.
 
-- `<Attribute>` (Optional, string, Defaults to `['*']`)
+If no value is specified, `attributesToRetrieve` uses the [`displayedAttributes` list](/reference/features/settings.md#displayed-attributes), which by default contains all attributes found in the documents.
 
-  Comma-separated list of attributes whose fields will be present in the returned documents.
+### Example
 
-  Defaults to to the [displayedAttributes list](/reference/features/settings.md#displayed-attributes) which contains by default all attributes found in the documents.
-
-#### Example
-
-If you want to get only the `overview` and `title` field and not the other fields, set `attributesToRetrieve` to `overview,title`.
+To get only the `overview` and `title` fields and ignore all other attributes, set `attributesToRetrieve` to `"overview", "title"`.
 
 <CodeSamples id="search_parameter_guide_retrieve_1" />
 
 ## Attributes to crop
```

Comment:
Hmmmm…that's a good point, but I think I'm a bit against it. I personally prefer docs whose structure mirrors that of what they're documenting. Users still needs to send the query like

```json
{
  "attributesToCrop": ["something"],
  "cropLength": 100
}
```

I'm afraid nesting them would establish a parent/child instead of a sibling relation.

Reply:
That is totally valid, thanks for your perspective! 🤝

Having re-read `cropLength`, I'm not sure how it behaves when `attributesToCrop` is not set. My guess would be that it uses the default crop of 200 bytes on either side of matches, and applies it to all `attributesToRetrieve`. Anyway, I just mention this because it would mean that there's a conceivable case for using only `cropLength` without `attributesToCrop`, which also supports your decision.

Reply:
Based on a quick search on a local instance, setting `cropLength` without configuring `attributesToCrop` has no effect. That said, it is still a valid query.

Perhaps we could add a sentence to `cropLength` stating that it is useless when used on its own?